src/Microsoft.ML.StandardLearners/Standard/LogisticRegression/LogisticRegression.cs

```diff
@@ -74,18 +74,16 @@ public sealed class Arguments : ArgumentsBase
         /// <param name="l2Weight">Weight of L2 regularizer term.</param>
         /// <param name="memorySize">Memory size for <see cref="LogisticRegression"/>. Low=faster, less accurate.</param>
         /// <param name="optimizationTolerance">Threshold for optimizer convergence.</param>
-        /// <param name="advancedSettings">A delegate to apply all the advanced arguments to the algorithm.</param>
-        public LogisticRegression(IHostEnvironment env,
+        internal LogisticRegression(IHostEnvironment env,
             string labelColumn = DefaultColumnNames.Label,
             string featureColumn = DefaultColumnNames.Features,
             string weights = null,
```

Comment:
i'd just delete it..

Reply:
I do see it being passed to base constructor `LbfgsTrainerBase`  where its used to initialize `WeightsColumn`   Thoughts ?


---
In reply to: [248543741](https://github.com/dotnet/machinelearning/pull/2163#discussion_r248543741) [](ancestors = 248543741)

Reply:
The ctor. Sorry for the bad selection. 

---
In reply to: [248769580](https://github.com/dotnet/machinelearning/pull/2163#discussion_r248769580) [](ancestors = 248769580,248543741)

Reply:
Deleting the constructor u mean ?  That would be #2100

---
In reply to: [248834382](https://github.com/dotnet/machinelearning/pull/2163#discussion_r248834382) [](ancestors = 248834382,248769580,248543741)